util/copycmd.bat

```diff
@@ -0,0 +1,6 @@
+REM @echo off
+set "PATH=C:\windows\system32;C:\windows;C:\windows\System32\Wbem"
+set "BASE=%1"
+set "DIR=%2"
+set "ARCHIVE=C:\test"
```

Comment:
Shouldn't this be `C:\data` rather than `C:\test`?

Reply:
It is run on a dedicated laptop rather than an instrument, this was just an example. It is more likely to be e:\somewhere as they will fit an external USB drive